system-tests/project-fixtures/angular/src/app/mount.cy.ts

```diff
@@ -343,6 +345,35 @@ describe("angular mount", () => {
     cy.get('img').should('be.visible').and('have.prop', 'naturalWidth').should('be.greaterThan', 0)
   })
 
+  it('should not override transient service', () => {
```

Comment:
Nit: Would be nice to wrap all of the `providers` tests in a describe/context as this file is getting long.

Reply:
Done and pushed my changes.

I also saw that when I run `eslint --fix` against that file, it re-formats more or less the whole content (also the existing tests). Shouldn't we do that?

Reply:
I believe we largely ignore the formatting of projects related to system-tests. I'm good with how your changes look now.